src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -91,6 +91,15 @@ const cspRulesDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
   return settings;
 };
 
+const mapManifestServiceUrlDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'map.manifestServiceUrl')) {
+    log(
+      `You should no longer use the map.manifestServiceUrl setting in kibana.yml to configure Elastic Maps Service settings. Use "map.emsTileApiUrl" and "map.emsFileApiUrl" instead.`
```

Comment:
Can we add that this is an undocumented setting, and that really, nobody should be configuring this (?).

e.g.:

> You should no longer use the map.manifestServiceUrl setting in kibana.yml to configure the location of the Elastic Maps Service settings. These settings have moved to the "map.emsTileApiUrl" and "map.emsFileApiUrl" settings instead. These settings are for development use only and should not be modified for use in production environments.